text/0000-ffi-safe-default.md

```diff
@@ -0,0 +1,77 @@
+- Feature Name: builtin-ffi-safe-only-as-default
+- Start Date: 2022-01-15
+- RFC PR:
+- Pony Issue:
+
+# Summary
+
+The current default is to allow FFI calls from any package. The optional `--safe` flag takes an argument of a colon delemeted list of packages and limits FFI calls to those and `builtin`.
+
+The purpose of this RFC is three-fold:
+
+* Change default from "allow all" to "deny all except stdlib"
+* Rename the `--safe` flag to the more explicit `--allow-ffi`
```

Comment:
What about --trusted? --trusted-packages?

Reply:
I prefer the explicit "allow-ffi", I don't think "trusted" as is explicit.